polyschedule/message_generators.py

```diff
@@ -0,0 +1,67 @@
+import asyncio
+from typing import Optional
+
+import rapidjson
+from vkbottle import PhotoMessageUploader
+
+import polyschedule.image_tools.image as image
+import polyschedule.keyboards as keyboards
+import polyschedule.schedule_api as schedule_api
+from polyschedule.image_tools.generate_algorythms import SimpleImageGenerateAlgorythm, CircleIconImageGenerateAlgorythm
+from polyschedule.exceptions import GettingScheduleError
+from config import api
+
+
+async def generate_day_message(day_date, group_id='35390') -> dict:
+    try:
+        day_schedule = await schedule_api.get_day(group_id, date=day_date)
+
+        if not day_schedule:
+            return {'message': 'На этот день расписания нет'}
+
+        uploader = await PhotoMessageUploader(api).upload(image.generate(day_schedule, CircleIconImageGenerateAlgorythm()))
+        payload = rapidjson.dumps({'day_date': day_date})
+
+        return {
+            'attachment': uploader, 
+            'keyboard': keyboards.MAIN, 
+            'payload': payload
+        }
+    except GettingScheduleError:
+        return {'message': 'Ошибка получения расписания с сервера Политеха'}
```

Comment:
Unfriendly error message

```suggestion
        return {'message': 'Что-то произошло с сервером Политеха, и поэтому мы не можем обработать Ваш запрос. Пожалуйста, попробуйте позже'}
```

When you send an error message to a user, make sure that user has a chance to understand theirs further steps to solve the error

Reply:
Try to avoid technical terms such as "error", make the message as friendly as possible.

Reply:
Good advice, thank you